tensorflow/lite/micro/testing/robot.resource.txt

```diff
@@ -9,7 +9,7 @@ Teardown With Custom Message
 Create Platform
     Execute Command           $logfile=@${UART_LOG}
     Execute Script            ${RESC}
-    Provides                  ready-platform
+    Provides                  ready-platform Reexecution
```

Comment:
Two+ spaces are required here. Robot syntax is whitespace-sensitive...

Reply:
thanks a lot! that fixed the issue locally.